AppKit/CPTabView.j

```diff
@@ -527,10 +531,7 @@ var CPTabViewDidSelectTabViewItemSelector           = 1 << 1,
 
 - (void)mouseDown:(CPEvent)anEvent
 {
-    var segmentIndex = [_tabs testSegment:[_tabs convertPoint:[anEvent locationInWindow] fromView:nil]];
-
-    if (segmentIndex != CPNotFound && [self selectTabViewItemAtIndex:segmentIndex])
-        [_tabs trackSegment:anEvent];
+    [_tabs trackSegment:anEvent];
```

Comment:
Unless there is a reason I don't see, it seems right to leave this events filtering.

Reply:
Well, if we leave the -selectTabViewItemAtIndex call, it will indeed select the tab view at mouseDown whereas one goal of the PR is to select it at mouseUp.

Reply:
No selection but you still need to filter out events that are not over a segment. Currently all mouse clicks on the content view are forwarded to the segmented control.

Reply:
Yes, indeed. I've considered that :

as the segmented control is called to test on which segment (or none) the click is done ;
as, if the click is done on a segment, the control segment will redo the same check on the trackSegment method ;
then it was OK to call trackSegment directly.
But you're right, this can be filtered. Moreover I would say this must be filtered because it introduces a disruption with Cocoa.

Thank you.